en/components/general/cli/getting-started-with-cli.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Step-by-Step Guide | Ignite UI for Angular | Infragistics
+_description: The Step-by-Step mode contains guided experience through the Ignite UI CLI options.
+_keywords: angular cli, ignite ui for angular, infragistics
+---
+
+## Getting Started with Ignite UI CLI
+
+>[!NOTE]
```

Comment:
I find this Note a bit strange. Maybe it will look better if it is just an introduction of 1-2 sentences. 
For example, 
If you’re creating a new Angular application from scratch, we recommend using the Ignite UI CLI as it will provide you with an user-friendly guided experience. The step-by-strep guide will lead you through all the setup options and your project will be scaffolded in a blink of an eye.
To get started, please, install Ignite UI CLI globally.